tests/test_events/test_api.py

```diff
@@ -248,15 +309,32 @@ def decorated_function() -> None:
         assert not EventAPI().pending_events
         assert not EventAPI().failed_events
 
-    def test_clears_queues_when_error_appeared_during_running_decorated_function(self) -> None:
+    def test_clears_pending_queue_when_error_appeared_during_running_decorated_function(
+        self,
+    ) -> None:
         @event_emitter
         def decorated_function() -> None:
             EventAPI().register(MyTestEvent({"x": 1}))
             raise RuntimeError
 
+        EventAPI().register(MyTestEvent({"x": 1}))
```

Comment:
Let's differentiate it to make it more obvious:
```suggestion
        EventAPI().register(MyTestEvent({"x": 2}))
```